modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -355,12 +354,12 @@ IECallContext::IECallContext(const IEUnit                                      &
                 return arg.get<cv::gimpl::RcDesc>().shape;
             });
 
-     for (const auto out_it : ade::util::indexed(outs)) {
-         // FIXME: Can the same GArg type resolution mechanism be reused here?
-         const auto port  = ade::util::index(out_it);
-         const auto desc  = ade::util::value(out_it);
-         m_results[port] = cv::gimpl::magazine::getObjPtr(m_res, desc);
-     }
+    for (const auto out_it : ade::util::indexed(outs)) {
+        // FIXME: Can the same GArg type resolution mechanism be reused here?
+        const auto port  = ade::util::index(out_it);
+        const auto desc  = ade::util::value(out_it);
+        m_results[port] = cv::gimpl::magazine::getObjPtr(m_res, desc);
+    }
```

Comment:
This code may not be correct

There is no 1-1 mapping between `output_objs` you get in the `InObjs` and the underlying operation's output ports.

Reply:
Probably we discussed that's this code work properly